dbt_coverage/__init__.py

```diff
@@ -486,6 +488,15 @@ def _new_miss_summary_row(self):
         return buf.getvalue()
 
 
+def check_manifest_version(manifest_json):
+    manifest_version = manifest_json['metadata']['dbt_schema_version']
+    if manifest_version != SUPPORTED_MANIFEST_SCHEMA_VERSION:
+        logging.warning(
+            "Unsupported manifest.json version \"%s\", unexpected behavior can occur. "
```

Comment:
@sweco I think it would make sense to link to the version matrix in here, if you think that's worth it 🙂

Reply:
Makes sense, thanks! 😊